pkgs/flox-cli/default.nix

```diff
@@ -61,14 +61,6 @@
       GIT_BIN = "${gitMinimal}/bin/git";
       NIX_BIN = "${nix}/bin/nix";
       PARSER_UTIL_BIN = "${parser-util}/bin/parser-util";
-      PKGDB_BIN =
-        if flox-pkgdb == null
-        then ""
```

Comment:
The default here should actually be `pkgdb` and 4 lines lower `flox-env-builder` since if you build `pkgdb` and `flox-env-builder` those binaries are then in the `PATH`

Reply:
it's pkgdb and flox-env-builder now